Externals/cubeb/src/cubeb_jack.cpp

```diff
@@ -986,7 +986,7 @@ cbjack_enumerate_devices(cubeb * context, cubeb_device_type type,
   if (!context)
     return CUBEB_ERROR;
 
-  uint32_t rate;
+  uint32_t rate = 0;
```

Comment:
this is somewhat risky, in the sense that it at least might add an operation compared to before